tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -134,26 +134,41 @@ - (void)addParentTo:(NSProgress*)child {
     ASSERT_EQ(YES, [progress isIndeterminate]);
 
     [progress setTotalUnitCount:5];
-    ASSERT_EQ(YES, [progress isIndeterminate]);
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setCompletedUnitCount:5];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
+
+    [progress setTotalUnitCount:0];
+    ASSERT_EQ(NO, [progress isIndeterminate]);
 
```

Comment:
thanks for the catch.
Yep EXPECT should be used here.

---
In reply to: [73764735](https://github.com/Microsoft/WinObjC/pull/836#discussion_r73764735) [](ancestors = 73764735,73764722)